src/Microsoft.AspNet.Mvc.TagHelpers/OptionTagHelper.cs

```diff
@@ -54,7 +54,7 @@ public override int Order
 
         /// <inheritdoc />
         /// <remarks>
-        /// Does nothing unless <see cref="FormContext.FormData"/> contains a
+        /// Does nothing unless <see cref="TagHelperContext.Items"/> contains a
         /// <see cref="SelectTagHelper.SelectedValuesFormDataKey"/> entry and that entry is a non-empty
```

Comment:
Throw `SelectTagHelper.SelectedValuesFormDataKey` away -- far away. This is an inefficient pattern that `Items` was designed to avoid. Instead use `typeof(SelectTagHelper)`.